opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/SpanBatchAdapter.java

```diff
@@ -85,12 +86,20 @@ private static Attributes createIntrinsicAttributes(SpanData span, Attributes at
 
   private static Attributes addPossibleErrorAttribute(SpanData span, Attributes attributes) {
     Status status = span.getStatus();
-    if (!status.isOk() && status.getDescription() != null && !status.getDescription().isEmpty()) {
-      attributes.put("error.message", status.getDescription());
+    if (!status.isOk()) {
+      attributes.put("error.message", getErrorMessage(status));
     }
     return attributes;
   }
 
+  private static String getErrorMessage(Status status) {
+    String description = status.getDescription();
+    if (description == null || description.isEmpty()) {
```

Comment:
What do you think about flipping the logic here?

Reply:
I think that I almost always try to prefer the non-inverted conditional because it requires less brainpower...especially when combined with a boolean!

Reply:
I get that. It reads funny to me right now, but I'm not 100% sure I can pin down why. How does it look with a ternary, rather than a full if block?

Reply:
The logic operator leaves it looking gross.  I'm going to factor out `isNullOrEmpty()` (because the other 2 of those that are available are marked "internal") and see what you think.